prdoc/pr_4936.prdoc

```diff
@@ -0,0 +1,11 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Balances Pallet: Emit events when TI is updated in currency impl
```

Comment:
```suggestion
title: "Balances Pallet: Emit events when TI is updated in currency impl"
```

Otherwise it is a nested TOML object.

Reply:
Should [this](https://github.com/paritytech/prdoc?tab=readme-ov-file#yaml-anchorshttps://github.com/paritytech/prdoc?tab=readme-ov-file#yaml-anchors) example be updated then?

Reply:
Yea possibly.